e2e/validate-schema.e2e-spec.ts

```diff
@@ -41,6 +43,8 @@ describe('Validate OpenAPI schema', () => {
       .addSecurityRequirements('bearer')
       .addSecurityRequirements({ basic: [], cookie: [] })
       .build();
+
+    Reflect.deleteMetadata("swagger/apiParameters", CatsController.prototype.getWithEnumParam);
```

Comment:
I don't know if there is a cleaner way to do this. The problem was, that on the second test case the metadata was merged and it was not the expected schema containing the enum values but the ref path to the enum schema which did not exist at that time. So the schema which was added on the second test was corrupt